frontend/src/views/BackendSessionExpiredView.vue

```diff
@@ -5,24 +5,24 @@
 </template>
 
 <script>
-import { AuthRoutes } from '@/utils/constants'
+import { mapState } from 'pinia'
+
 import { useAuthStore } from '@/stores/auth'
-import { mapActions } from 'pinia'
+import { AuthRoutes } from '@/utils/constants'
+
 export default {
   name: 'BackendSessionExpiredView',
   data() {
     return {
-      link: AuthRoutes.SESSION_EXPIRED,
+      link: '/stuff',
```

Comment:
is it really '/stuff'?

Reply:
Wow. I sure messed that one up.

Reply:
I was messing around with removing the link which can be done.